internal/core/entity/entity.go

```diff
@@ -93,7 +99,9 @@ func (info *BaseInfo) Updating(storedInfo *BaseInfo) {
 
 func (info *BaseInfo) KeyCompat(key string) {
 	if info.ID == "" && key != "" {
-		info.ID = key
+		// TODO: we should get id by prefix in config
+		arr := strings.Split(key, "/")
```

Comment:
We should use $prefix/routes/$name. Maybe we can refactor it to a common method?

Reply:
It is difficult to have an elegant implementation under the current code structure. I will first give a compromise solution, so that encode and decode are distributed in different modules

Reply:
OK, let's add an issue to track it.